server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/AnalyzeVisitor.java

```diff
@@ -417,8 +424,9 @@ private List<Pair<Expression, String>> analyzeSelect(
           break;
         }
       }
+      outputExpressionMap.put(columnIndex, outputExpressions);
```

Comment:
I think the columnIndex is very useful in later use, so it's better to set from 0, not 1. If it begins at 0, we can easily get the corresponding ResultColumn by `queryStatement.getSelectComponent().getResultColumns().get(columnIndex)`

Reply:
ok~~